monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -0,0 +1,33 @@
+package com.gu.support.workers.lambdas
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import org.joda.time.DateTime
+import com.amazonaws.services.lambda.runtime.Context
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.generic.auto._
+import com.gu.config.Configuration
+import com.gu.emailservices.{ThankYouEmailService, ThankYouFields}
+import com.gu.support.workers.model.CreateZuoraSubscriptionState
+
+
+class SendThankYouEmail(
+  thankYouEmailService: ThankYouEmailService = new ThankYouEmailService(Configuration.emailServicesConfig.thankYouEmailQueue)
+) extends FutureHandler[CreateZuoraSubscriptionState, Unit] with LazyLogging {
+
+  override protected def handlerFuture(state: CreateZuoraSubscriptionState, context: Context): Future[Unit] = {
+    logger.info(s"state: $state")
+    sendEmail(state)
+  }
+
+  def sendEmail(state: CreateZuoraSubscriptionState): Future[Unit] = {
+    thankYouEmailService.send(ThankYouFields(
+      email = state.user.primaryEmailAddress,
+      created = DateTime.now(),
+      amount = state.amount,
+      currency = "GBP",
```

Comment:
I think it would be good to pass this through in the State objects so we can add new ones more easily in future. We can make a separate PR for that though as it is used in the Zuora lambda as well.